docs/ProductSummaryList.md

```diff
@@ -27,29 +27,42 @@ This block is used to specify what variation of `product-summary` to be used to
-| ---- | ----- | ----------- |
+| Name            | Value             | Description                                                                                                                                            |
+| --------------- | ----------------- | ------------------------------------------------------------------------------------------------------------------------------------------------------ |
 | First Available | `FIRST_AVAILABLE` | Most performant, ideal if you do not have a SKU selector in your shelf. Will return only the first available SKU for that product in your shelf query. |
-| All Available | `ALL_AVAILABLE` | A bit better performace, will only return SKUs that are available, ideal if you have a SKU selector but still want a better performance. |
-| All | `ALL` | Returns all SKUs related to that product, least performant option. |
+| All Available   | `ALL_AVAILABLE`   | A bit better performace, will only return SKUs that are available, ideal if you have a SKU selector but still want a better performance.               |
+| All             | `ALL`             | Returns all SKUs related to that product, least performant option.                                                                                     |
 
 For `InstallmentCriteriaEnum`:
 
-| Name | Value | Description |
-| ---- | ----- | ----------- |
-| Maximum without interest | `MAX_WITHOUT_INTEREST` | Will display the maximum installment option with no interest. |
-| Maximum | `MAX_WITH_INTEREST` | Will display the maximum installment option having interest or not. |
+| Name                     | Value                  | Description                                                         |
+| ------------------------ | ---------------------- | ------------------------------------------------------------------- |
+| Maximum without interest | `MAX_WITHOUT_INTEREST` | Will display the maximum installment option with no interest.       |
+| Maximum                  | `MAX_WITH_INTEREST`    | Will display the maximum installment option having interest or not. |
+
+For `PreferredSKUEnum`:
+
+| Name            | Value             | Description                                        |
+| --------------- | ----------------- | -------------------------------------------------- |
+| First Available | `FIRST_AVAILABLE` | Selects the first available SKU in stock it finds. |
+| Last Available  | `LAST_AVAILABLE`  | Selects the last available SKU in stock it finds.  |
+| Cheapest        | `PRICE_ASC`       | Selects the cheapest SKU in stock it finds.        |
+| Most Expensive  | `PRICE_DESC`      | Selects the most expensive SKU in stock it finds.  |
+
+⚠️ Keep in mind that the default behaviour for the initially selected SKU, has now changed to a Product (field) specification.  
+If that one doesn't exist, it will use the `preferredSKU` prop. You can read more about it, and how to implement it in [Recipes](https://vtex.io/docs/recipes/all)
```

Comment:
Is there a recipe for this already? If so, update this link to point directly to the recipe, instead of to the recipe list.

Also, I think we're supposed to be linking to developers.vtex.com instead of vtex.io/docs 😄

Reply:
The education team will create the recipe once this is approved.
Regarding the url, I don't see the Recipes subdirectory in developers.vtex.com

Reply:
Then I think you could remove this part for now